src/main/java/uk/gov/companieshouse/web/companydocuments/service/CompanyService.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.companieshouse.web.companydocuments.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+import uk.gov.companieshouse.web.companydocuments.api.ApiClientService;
+import uk.gov.companieshouse.web.companydocuments.exception.ServiceException;
+import uk.gov.companieshouse.web.companydocuments.model.CompanyDetail;
+import uk.gov.companieshouse.web.companydocuments.transformer.CompanyTransformer;
+
+@Service
+public class CompanyService {
+
+    @Autowired
+    private ApiClientService apiClientService;
+
+    @Autowired
+    private CompanyTransformer companyTransformer;
```

Comment:
Can we start to use constructor injection instead of field injection? It's better practice:
https://reflectoring.io/constructor-injection/